site/gatsby-site/server/fields/common.ts

```diff
@@ -90,140 +88,82 @@ export interface UserAdminData {
     userId?: string;
 }
 
-export const getUserAdminData = async (userId: string) => {
-
-    const userApiResponse = await apiRequest({ path: `/users/${userId}` });
-
-    const response: UserAdminData = {};
-
-    if (userApiResponse.data) {
-
-        response.email = userApiResponse.data.email;
-        response.creationDate = new Date(userApiResponse.creation_date * 1000);
-        response.lastAuthenticationDate = new Date(userApiResponse.last_authentication_date * 1000);
-        response.disabled = userApiResponse.disabled;
-    }
-
-    return response;
-}
-
-export const getUsersAdminData = async () => {
+const usersCache: UserAdminData[] = [];
```

Comment:
Document this global value please... it's getting mutated by a "get" function, which is indirect. Maybe `getAndCacheUserAdminData()` is better.

Reply:
I moved the cache to the script itself to avoid keeping a global cache, as you mentioned.